rpc/gnmi/gnmi-specification.md

```diff
@@ -898,6 +900,15 @@ accuracy for individual data items, the `Subscribe` RPC is recommended to
 request a telemetry stream (see [Section
 3.5.2](#352-sending-telemetry-updates)).
 
+### 3.3.4 GetResponse Behavior Table
+
+| GetRequest Scenario                                                                                                      | Target Behavior |
+| ------------------------------------------------------------------------------------------------------------------------ | ---------------- |
+| Subscribed paths exist or a YANG default value is [in use](https://datatracker.ietf.org/doc/html/rfc7950#section-7.6.1). | Value(s) are returned |
+| Subscribed paths are syntactically correct but one or more paths do not (yet) exist.                                     | Return `NOT_FOUND` |
```

Comment:
should it further clarify here as `... do not (yet) exist or default value is not in use`?

Reply:
Thanks, expanded sentence in both tables.